addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js

```diff
@@ -279,3 +279,41 @@ ProductScreen.check.totalAmountIs('1.00');
 PosLoyalty.check.isRewardButtonHighlighted(false);
 
 Tour.register('PosLoyaltyTour10', { test: true, url: '/pos/web' }, getSteps());
+
+startSteps();
+
+ProductScreen.do.clickHomeCategory();
+ProductScreen.do.confirmOpeningPopup();
+
+ProductScreen.do.clickPartnerButton();
+ProductScreen.do.clickCustomer('AAA Partner');
+ProductScreen.do.clickDisplayedProduct('Product Test');
+ProductScreen.do.pressNumpad('3')
```

Comment:
```suggestion
ProductScreen.exec.addOrderline('Product Test', '3');
```
Use addOrderline. It waits for the orderline to be rendered properly (with correct quantity), before going to the next step.